### ISSUE TITLE
System.AggregateException incluided in AutoMapper 4.0.4

### ISSUE THREAD

**Issue by nberardi** [AutoMapper/AutoMapper#825]
This is causing the following error in any source code that already references an AggregateException since it is already included in mscorlib.  

```
Error CS0433: The imported type `System.AggregateException' is defined multiple times (CS0433) (test)
```

It would be great if AutoMapper just used the mscorlib version.

**Comment by lbargaoanu** [comment]
#823?

**Comment by jbogard** [comment]
Pretty sure, @nberardi what platform are you targeting here?

**Comment by nberardi** [comment]
Xamarin.Mac 2.0 Mobile Framework build. Not the .NET 4.5 build. It uses the same libraries and compiler as Xamarin.iOS. 

I just looked through the commit history. It looks like you fixed the bug in 4.1.0 but it hasn't been shipped to NuGet yet. https://github.com/AutoMapper/AutoMapper/commit/b336d11b177c2f006666c27d2eb1b96d21d427df

Also there is another bug with Xamarin.Mac. It tries to includes a whole bunch libraries that don't exist in the mobile framework version. This isn't the end of the world. But just more of an annoyance that I have to go through and clean up. 

I was looking for your nuspec so I could see if the problem was an obvious one. But I didn't see it. 

Xamarin.Mac is defaulting to your portable build. 

--
Nick Berardi
(484) 302-0125
Sent on the go from my phone.
